src/main/resources/hudson/plugins/active_directory/ActiveDirectorySecurityRealm/help-bindName.html

```diff
@@ -10,4 +10,8 @@
     <p>
     This field must be the full user principal name with domain name, like "joe@europe.contoso.com", or
     a LDAP-style distinguished name, such as "CN=Joe Chin,OU=europe,DC=contoso,DC=com".
+
+    <p>
+    Active Directory requires that the <a href="http://social.technet.microsoft.com/wiki/contents/articles/5312.active-directory-characters-to-escape.aspx">following ten characters</a>
+    be escaped with the backslash "\" escape character if they appear in any of the individual components of a distinguished name.
```

Comment:
So this actually doesn't need to be a DN - it can also be a `UserPrincipalName` or `DOMAIN\user`

Slightly OT I know :-)